libs/platform/table/components/table-cell-header-popover/table-cell-header-popover.component.html

```diff
@@ -99,6 +99,7 @@
                     fd-form-control
                     [id]="'fdp-table-column-filtering-' + column.name"
                     (keydown.enter)="_setColumnHeaderFilterBy(column.key, $any($event.target)?.value)"
+                    (keydown)="$event.stopPropagation()"
```

Comment:
I am confused why this works but it does

Reply:
The second (keydown) event listener with stopPropagation() ensures that the keydown event does not bubble up the DOM tree. This prevents unintended behavior such as triggering other event listeners higher in the component hierarchy that could interfere with the expected behavior of input field. Without this, the keydown event might propagate to other components, such as the popover or list items, causing the input field to lose focus or trigger other actions.

Reply:
I think leave a comment here explaining the situation, I'd be confused if I stumbled upon this without contrext

Reply:
<img width="1030" alt="image" src="https://github.com/user-attachments/assets/3bec3b1a-80b1-4b0f-a07b-767401881521">